R/config.R

```diff
@@ -251,61 +243,6 @@ validate_config_task_id_text <- function(webevals_config, task_groups, task_id_n
 }
 
 
-#' Get a data frame with 1 row for each metric, matching the metric with the
```

Comment:
`get_metric_name_to_output_type` and `get_standard_metrics` moved to `utils-metrics.R`.  I'm going to need these functions in the code to compute scores, so it's better to have them in a `utils` file.